src/app/patient-summary/page.tsx

```diff
@@ -0,0 +1,20 @@
+import Box from "@mui/material/Box";
+import { Container } from "@mui/material";
+import PatientSummary from "./components/PatientSummary";
+
+export default function PatientSummaryPage() {
+  return (
+    <Container maxWidth={false}>
+      <Box
+        display="flex"
+        flexDirection="column"
+        alignItems="center"
+        justifyContent="center"
+        paddingTop={8}
+        paddingBottom={8}
+      >
+        <PatientSummary />
```

Comment:
Suggestion: Maybe not for now, but what would be great is if we could have the sample FHIR bundle retrieved from this page, and the json supplied to the PatientSummary as a prop. This means we can just supply the component with json which it will render.

Later on, instead of a static JSON object, this will dynamically be fetched which we will then supply to the component to render the data

Reply:
Yes, that's a good point. I've refactored it accordingly.